Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1933,7 +1933,7 @@ - (void)_recalculateViewportSizesWithMinimumViewportInset:(CocoaEdgeInsets)minim
     auto frame = WebCore::FloatSize(self.frame.size);
 
 #if PLATFORM(MAC)
-    auto additionalInsets = _impl->obscuredContentInsets();
+    auto additionalInsets = _impl->effectiveObscuredContentInsetsConsideringAttachedInspector();
```

Comment:
```suggestion
    auto additionalInsets = _impl->effectiveObscuredContentInsetsConsideringAttachedInspector();
```

I think that either `effectiveObscuredContentInsets` would be sufficient, or `obscuredContentInsetsAdjustedForAttachedInspector`.